example/test_problem/Hydro/ClusterMerger/Input__TestProb

```diff
@@ -1,32 +1,68 @@
 Merger_Coll_PosY2       7500.0    # Y-center of cluster 2 in kpc
 Merger_Coll_VelX2       -1125.0000000000002    # X-velocity of cluster 2 in km/s
 Merger_Coll_VelY2       0.0    # Y-velocity of cluster 2 in km/s
 Merger_Coll_IsGas2      1 # If cluster 2 should have gas and not have DM only
+Bondi_MassBH2           3.4e8 # black hole mass (in Msun)
+Mdot_BH2                0.0 # accretion rate of cluster 2 in Msun/yr
+Jet_HalfHeight2         0.70 # jet 2: half height of the cylinder-shape jet source (in kpc) 
+Jet_Radius2             0.35 # jet 2: radius of the cylinder-shape jet source (in kpc) 
 
 # parameters for cluster 3 (disabled for this example)
 Merger_File_Prof3       none # profile table of cluster 3
 Merger_File_Par3        none # particle file of cluster 3
+Unit_R3                 1.0 # the unit of length in cluster 3's IC file (in cgs)           
+Unit_D3                 1.0 # the unit of density in cluster 3's IC file (in cgs)          
+Unit_P3                 1.0 # the unit of pressure in cluster 3's IC file (in cgs) 
 Merger_Coll_PosX3       0 # X-center of cluster 3 in kpc
 Merger_Coll_PosY3       0.0 # Y-center of cluster 3 in kpc
 Merger_Coll_VelX3       0.0 # X-velocity of cluster 3 in km/s
 Merger_Coll_VelY3       0.0 # Y-velocity of cluster 3 in km/s
 Merger_Coll_IsGas3      0 # If cluster 3 should have gas and not have DM only
+Bondi_MassBH3           3.4e8 # black hole mass (in Msun)
+Mdot_BH3                0.0 # accretion rate of cluster 3 in Msun/yr
+Jet_HalfHeight3         0.70 # jet 3: half height of the cylinder-shape jet source (in kpc) 
+Jet_Radius3             0.35 # jet 3: radius of the cylinder-shape jet source (in kpc) 
 
-# other parameters
-Merger_Coll_UseMetals   0 # look for a metal field in the profile files
+# parameters of AGN jet feedback
+Accretion_Mode          1 # 1: hot mode; 2: code mode; 3: combine (hot + cold)
+eta                     1.0 # mass loading factor in jet feedback
+eps_f                   0.001 # the radiative efficiency in jet feedback
+eps_m                   0.0 # the fraction of total energy that goes into the thermal energy in jet feedback
+R_acc                   4.0 # accretion radius: compute the accretion rate (in kpc)
+R_dep                   4.0 # radius to deplete the accreted gas (in kpc)   
+JetDirection_case       1 # Methods for choosing the jet direction: 1. Fixed at x-axis; 2. Import from table (generate JetDirection.txt); 3. Align with angular momentum
+
+# other	parameters
+Merger_Coll_UseMetals   0 # look for a metal field in the profile files [1]
+Merger_Coll_LabelCenter 1 # label the particle closest to the center of each cluster [1]
+AdjustBHPos             1 # (true/false) --> Adjust the BH position
+AdjustBHVel             0 # (true/false) --> Adjust the BH velocity
+AdjustPeriod            10 # the time interval of adjustment (in Myr)
+fixBH                   0 # fix the BH at the simulation box center and set its velocity to be zero (1 cluster only)
```

Comment:
The `ClusterMerger` test problem technically allows for a single cluster to be placed anywhere in the domain, so in theory `fixBH` should not go to the center of the box but to wherever we've placed the cluster to be. But this is a minor issue that could be fixed later.